pro/src/ui-kit/form/PasswordInput/PasswordInput.tsx

```diff
@@ -11,39 +17,75 @@ interface IPasswordInputProps {
   label: string
   name: string
   placeholder: string
+  withErrorPreview?: boolean
 }
 
 const PasswordInput = ({
   label,
   name,
   placeholder,
+  withErrorPreview = false,
 }: IPasswordInputProps): JSX.Element => {
   const [isPasswordHidden, setPasswordHidden] = useState(true)
+  const [field] = useField({ name })
+  const [errors, setErrors] = useState<{ [key: string]: boolean }>({})
+  const [displayLocalErrors, setDisplayLocalErrors] = useState(false)
+
+  useEffect(() => {
+    if (withErrorPreview) {
+      // We only handle locally the password requirements.
+      // The other errors are handled by the FieldLayout.
+      setDisplayLocalErrors(withErrorPreview && field.value.length > 0)
+      setErrors(passwordValidationStatus(field.value))
+    }
+  }, [withErrorPreview, field.value])
 
   const handleToggleHidden = (e: React.MouseEvent<HTMLElement>) => {
     e.preventDefault()
     setPasswordHidden(currentSetPasswordHidden => !currentSetPasswordHidden)
   }
 
   return (
-    <div className={styles['password-input-wrapper']}>
-      <TextInput
-        className={styles['password-input']}
-        label={label}
-        name={name}
-        placeholder={placeholder}
-        type={isPasswordHidden ? 'password' : 'text'}
-        rightButton={() => (
-          <button onClick={handleToggleHidden}>
-            {isPasswordHidden ? (
-              <IcoEyeClose className={styles['password-input-eye-ico']} />
-            ) : (
-              <IcoEyeOpen className={styles['password-input-eye-ico']} />
-            )}
-          </button>
-        )}
-      />
-    </div>
+    <>
+      <div className={styles['password-input-wrapper']}>
+        <TextInput
+          className={styles['password-input']}
+          label={label}
+          name={name}
+          placeholder={placeholder}
+          type={isPasswordHidden ? 'password' : 'text'}
+          hideFooter={displayLocalErrors}
+          rightButton={() => (
+            <button onClick={handleToggleHidden}>
+              {isPasswordHidden ? (
+                <IcoEyeClose className={styles['password-input-eye-ico']} />
+              ) : (
+                <IcoEyeOpen className={styles['password-input-eye-ico']} />
+              )}
+            </button>
+          )}
+        />
+      </div>
+      {displayLocalErrors && (
```

Comment:
```suggestion
        />
       {displayLocalErrors && <ValidationMessageList />
      </div>
```
Ca éviterait d'avoir à utiliser un fragment, même si ca fonctionne tres bien avec :)